### DISCUSSION TITLE
albumart when using upnp?

### DISCUSSION THREAD

**Discussion by kreativkod** [rern/rAudio#1251] [Feature Requests]
Request: album art when using UPnP

A while ago I tried several versions of rAudio, using BubbleUPnP to play Qobuz files. It works but rAudio does not show album art although it does show titles and albums and artist data. I thought album art was simply out of reach in this kind of solution and was prepared to live without it, but then I tried MoodeAudio and the same setup worked out of the box showing album art without fuzz.

No, I will not switch to Moode because I don't like the browsing. 

But it would be soooo nice to have the album art in rAudio !!

Can this be arranged? Updating the config for upmpdcli or something?

**Comment by audiochrome** [comment]
actually it's the only negative point, I hope it will be implemented in the future

**Comment by kreativkod** [comment]
It's been a while since I opened this topic, so I ping you @rern in case you have not seen this. 
Or is it maybe already in the pipeline?

**Comment by R101G** [comment]
No offence intended @kreativkod but, unlike Moode, rAudio is pretty much the work of one genius. I would much rather that there were no new features until what we already have is bug-free, and working as smoothly as it used to!

**Comment by Maikeleg** [comment]
I would very much appreciate this functionality too!
I'm using upnp because it sounds better, the reason I chose to use rAudio (and learn Arch-linux)
As upnp server I use Minimserver, as client the Linn app
